images/alpine.yaml

```diff
@@ -241,6 +270,26 @@ files:
     # Stuff to do before rebooting
     ::shutdown:/sbin/openrc shutdown
 
+- path: /etc/init.d/lxd-agent-9p
+  generator: dump
+  content: |-
+    #!/sbin/openrc-run
+
+    description="LXD - agent - 9p mount"
+    command=/bin/mount
+    command_args="-t 9p config /run/lxd_config/9p -o access=0,trans=virtio"
+
+    depend() {
+      want udev-trigger
+      after udev-trigger
+    }
```

Comment:
This might be specific to Alpine. The only other distro using openRC is Gentoo which I will look at next. If Gentoo needs this as well, I will add it to the lxd-agent generator.

Reply:
We have that generator now so we should use it here.